lib/registries/addon/components/registries-tags-widget/template.hbs

```diff
@@ -9,6 +9,7 @@
     @placeholder={{t 'osf-components.tags-widget.add_tag'}}
     @readOnly={{this.readOnly}}
     @maxlength={{1024}}
+    @showRemoveButtons={{false}}
```

Comment:
given this, let's remove the `@removeTagAtIndex={{action this.removeTag}}` above